vendor/assets/stylesheets/susy/susy/language/susy/_background.scss

```diff
@@ -0,0 +1,386 @@
+// Background Grid Syntax
+// ======================
+
+$susy-overlay-grid-head-exists: false;
+
+
+// Show Grid/s
+// -----------
+// Show grid on any element using either background or overlay.
+// - [$grid]  : <settings>
+@mixin show-grid(
+  $grid: $susy
+) {
+  $inspect: $grid;
+  $_output: debug-get(output, $grid);
+
+  @include susy-inspect(show-grid, $inspect);
+  @if $_output == overlay {
+    @include overlay-grid($grid);
+  } @else {
+    @include background-grid($grid);
+  }
+}
+
+@mixin show-grids(
+  $grid: $susy
+) {
+  @include show-grid($grid);
+}
+
+// Background Grid
+// ---------------
+// Show a grid background on any element.
+// - [$grid]  : <settings>
+@mixin background-grid(
+  $grid: $susy
+) {
+  $inspect  : $grid;
+  $_output  : get-background($grid);
+
+  @if length($_output) > 0 {
+    $_flow: susy-get(flow, $grid);
+
+    $_image: ();
+    @each $name, $layer in map-get($_output, image) {
+      $_direction: if($name == baseline, to bottom, to to($_flow));
+      $_image: append($_image, linear-gradient($_direction, $layer), comma);
+    }
+    $_output: map-merge($_output, (image: $_image));
+
+    @include background-grid-output($_output...);
+    @include susy-inspect(background-grid, $inspect);
+  }
+}
+
+
+// Overlay Grid
+// ------------
+// Generate an icon to trigger grid-overlays on any given elements.
+// $grids...  : <selector> [<settings>] [, <selector>]*
+@mixin overlay-grid (
+  $grid: $susy
+) {
+  @if not($susy-overlay-grid-head-exists) {
+    @at-root head { @include overlay-head($grid); }
+    @at-root head:before { @include overlay-trigger; }
+    @at-root head:hover { @include overlay-trigger-hover; }
+    $susy-overlay-grid-head-exists: true !global;
+  }
+
+  head:hover ~ &,
+  head:hover ~ body & {
+    position: relative;
+    &:before {
+      @include grid-overlay-base;
+      @include background-grid($grid);
+    }
+  }
+}
+
+
+// [Private] Overlay Trigger
+// -------------------------
+@mixin overlay-trigger {
+  content: "|||";
+  display: block;
+  padding: 5px 10px;
+  font: {
+    family: sans-serif;
+    size: 16px;
+    weight: bold;
+  }
+}
+
+
+// [Private] Overlay Trigger Hover
+// -------------------------------
+@mixin overlay-trigger-hover {
+  background: rgba(white, .5);
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.